app/src/components/layout/Navbar/Navbar.tsx

```diff
@@ -25,30 +26,39 @@ import { ConfigurableLogo, DefaultNavLogo } from '../LogoComponents';
 
 const navItems = [
   {
-    title: 'home',
+    title: 'Home',
     path: '/',
     key: 'home',
   },
   {
-    title: 'blocks',
+    title: 'Blocks',
     path: '/blocks',
     key: 'blocks',
   },
   {
-    title: 'peers',
+    title: 'Peers',
     path: '/peers',
     key: 'peers',
   },
   {
-    title: 'validators',
+    title: 'Validators',
     path: '/validators',
     key: 'validators',
   },
 ];
 
 export const Navbar: React.FC = () => {
+  const [isOpened, setIsOpened] = useState<boolean>();
+
+  const [selectedRoute, setSelectedRoute] = useState(null);
```

Comment:
I find it odd that this is typed as `null`. What happens when the user first visits the page, before selecting any nav item manually? How does the selected state get set to any nav item without the user clicking on the item?

Reply:
It's now and empty string

Reply:
Alright, but is the nav item set to the current page when the app first loads? Without the user clicking on the nav item, that is.

Reply:
@Thistle-Strum It defaults to home no?

Reply:
@DorianKwan 
Made that change :)